### PR TITLE
[17.0][MIG] mail_tracking_mass_mailing: Migration to 17.0

### PR DESCRIPTION
**Pull request by chienandalu**
- Standard migration and some cleanup

cc @Tecnativa TT49915

please review @pedrobaeza @victoralmau

### CODE REVIEW

File: mail_tracking_mass_mailing/__manifest__.py [hunk 1]
```diff
@@ -7,13 +7,11 @@
...
     "name": "Mail tracking for mass mailing",
     "summary": "Improve mass mailing email tracking",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
     "category": "Social Network",
     "website": "https://github.com/OCA/social",
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "application": False,
-    "installable": True,
```
Comment (pedrobaeza) [inline mail_tracking_mass_mailing/__manifest__.py]: Don't remove this key

File: pandoc-3.1.13-1-amd64.deb
Comment (pedrobaeza) [inline pandoc-3.1.13-1-amd64.deb:1]: Although the migration guide says so, let's not remove this, as we will soon do a forced push to remove them.

**Review by pedrobaeza — APPROVED** [review]

**Review by pilarvargas-tecnativa — APPROVED** [review]
:+1:

**Review by victoralmau — APPROVED** [review]

**Comment by chienandalu** [comment]
Changes done

**Pull request merged**